.github/workflows/ci.yml

```diff
@@ -0,0 +1,200 @@
+name: CI
+on: [pull_request]
```

Comment:
Should we change this to every `push`?

Reply:
That's what I did initially, but then it doesn't show up as a CI check 🤔

Reply:
Oh that's odd! Maybe because it's not yet on the main branch?

Reply:
It seems like it's designed like that: https://github.community/t/github-actions-as-status-checks/16666/2

Reply:
Oh should add both push and pull_request or will that create two runs for each PR push? Seems bad to miss out on ci on main branch pushes/merges

Reply:
Although for updater we set it to `push` and shows up as a check https://github.com/dependabot/dependabot-updater/pull/992#partial-pull-merging

Reply:
Ah you're right! Ok let's set it to `push`, probably because it's not on main like you mentioned